### DISCUSSION TITLE
Does formic have support for joi schema

### DISCUSSION THREAD

**Discussion by i0000010** [jaredpalmer/formik#3912] [Q&A]
I read in the Formic documentation that there is support for Yup schema validation using the validationSchema prop. Can I pass a Joi schema instead? Considering that Joi now has an official browser build, it would be extremely convenient to reuse the same Joi schema used to build my rest API. I can't find any mention of support for Joi schema in the documentation.

**Comment by cgarcia369** [comment]
You can use npm joi-formik-adapter package https://www.npmjs.com/package/joi-formik-adapter
